sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosResponse.java

```diff
@@ -127,6 +156,10 @@ public CosmosDiagnostics getDiagnostics() {
      * @return end-to-end request latency for the current request to Azure Cosmos DB service.
      */
     public Duration getDuration() {
+        if (resourceResponseWrapper == null) {
+            return null;
```

Comment:
This is a breaking change. Currently if the response is null, we return Duration.ZERO. Can we please revert this?

Reply:
Let's discuss on Monday - I wont' change it right now because I don't agree that it is a breaking change - it would have thrown a NullPointerException here - I only added this to avoid the NullPointerException. If you still think this is considered breaking I can add an internal property - but seems ok to do that in a separate PR or within the retriable write PR after we discussed on Monday.